.scripts/publish_client_pypi.sh

```diff
@@ -3,13 +3,13 @@
 # WARNING: DO NOT EDIT!
 #
 # This file was generated by plugin_template, and is managed by it. Please use
-# './plugin-template --travis pulp_npm' to update this file.
+# './plugin-template --github pulp_npm' to update this file.
 #
 # For more info visit https://github.com/pulp/plugin_template
 
 set -euv
 
-pip install twine setuptools wheel
+pip install twine
 
 cd "${GITHUB_WORKSPACE}"
 export REPORTED_VERSION=$(http pulp/pulp/api/v3/status/ | jq --arg plugin pulp_npm -r '.versions[] | select(.component == $plugin) | .version')
```

Comment:
90% of these two bindings gen scripts are identical, it'd be nice if we could abstract that.

Reply:
Actually i experienced a problem with publishing one artifact and then being unable to generate the next one. The release was broken beyond repair.

So it would be good, if we could first generate all packages and then start publishing them. I think that leads to not abstracting, but merging those scripts.

Reply:
@mdellweg that sounds like a good idea. would you file an issue?